target/linux/generic/files/drivers/mtd/mtdsplit/mtdsplit_uimage.c

```diff
@@ -407,6 +407,43 @@ static struct mtd_part_parser uimage_fonfxc_parser = {
 	.parse_fn = mtdsplit_uimage_parse_fonfxc,
 };
 
+/**************************************************
+ * ubootpad96
+ **************************************************/
+
+#define ubootpad96_PAD_LEN		96
```

Comment:
Create separate commit for this changes with proper description. @rmilecki can you check this?

Reply:
Do you mean a separate commit in the same branch/pull request, or a new pull request?

Reply:
Yes, this is good, thanks